### PR TITLE
Dynamic font size on connection manual screen

### LINKED ISSUE #5518: [Mobile Payments] Support for Dynamic Type in connection instructions screen.
Outcome of #4348

Support for Dynamic Type in the the reader connection tutorial can be improved:

| Text 135% | Text 310% |
| ---- | ---- |
|  |  |

pdfdoF-hD-p2

### PR DESCRIPTION
**Pull request by kidinov**
Closes: #5518 

### Description

Adds support for dynamic font size on the connection instruction screen

### Testing instructions

* Go to the settings
* IPP
* Manage to card readers
* Via accessibility change font size and notice that it reflected on the instruction screen

### Screenshots

image
image

---
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

**Review by jaclync — COMMENTED** [review]
It looks like the dynamic type change was on the "Learn more" text? While the "Learn more" text scales with the device font size, the list of instructions above somehow gets truncated. I was wondering if we can just let the view be scrollable like in landscape in this case?

`trunk` | PR branch
-- | --
Simulator Screen Shot - iPhone 11 - 2022-05-20 at 09 07 45 | Simulator Screen Shot - iPhone 11 - 2022-05-20 at 09 06 18

**Comment by kidinov** [comment]
@jaclync I've added a workaround for this case 45f6e819f8ac380279bfecb47c0f137fda93d2b8

How does this look?

File: WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReadersV2/CardReaderSettingsSearchingViewController.swift [hunk 1]
```diff
@@ -112,13 +112,20 @@ struct CardReaderSettingsSearchingView: View {
...
         get {
             verticalSizeClass == .compact
         }
     }
 
+    var isSizeCategoryLargeThanExtraLarge: Bool {
+        get {
+            sizeCategory >= .accessibilityMedium
+        }
+    }
```
Comment (jaclync) [inline WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReadersV2/CardReaderSettingsSearchingViewController.swift]: nit:
```suggestion
    var isSizeCategoryLargeThanExtraLarge: Bool {
        sizeCategory >= .accessibilityMedium
    }
```

**Review by jaclync — APPROVED** [review]
Thanks for the update! It looks great nice now :shipit:

**Pull request merged**